test/gst-va/decode/10bit/av1.py

```diff
@@ -23,12 +23,16 @@ def before(self):
   def test(self, case):
     vars(self).update(spec[case].copy())
 
-    dxmap = {".ivf" : "ivfparse", ".webm" : "matroskademux", ".mkv" : "matroskademux"}
+    dxmap = {".ivf" : "ivfparse", ".webm" : "matroskademux", ".mkv" : "matroskademux", ".obu" : "av1parse"}
     ext = os.path.splitext(self.source)[1]
-    assert ext in dxmap.keys(), "Unrecognized source file extension {}".format(ext)
+    dx = dxmap.get(ext, None)
+    assert dx is not None, "Unrecognized source file extension {}".format(ext)
+
+    if "av1parse" not in dx:
+      dx += " ! av1parse"
 
     vars(self).update(
       case        = case,
-      gstdecoder  = "{} ! av1parse ! vaav1dec".format(dxmap[ext]),
+      gstdecoder  = "{} ! vaapiav1dec".format(dx),
```

Comment:
Aha, yes... wrong av1d plugin got introduced here.  I'm glad you found this and fixed it in #311

Reply:
It's usually best to wait for at least 1 person to give a review before merge.

Reply:
> It's usually best to wait for at least 1 person to give a review before merge.

Thanks, Yes, it is better to follow this step.